README.md

```diff
@@ -56,10 +56,13 @@ python -m pip install -r PySolid/requirements.txt
 
 ```bash
 # option 1: use pip to install pysolid into the current environment
-# use "pip install -e" to install in the development mode
 python -m pip install PySolid
 
-# option 2: manually compile the Fortran code and setup environment variable
+# option 2: use pip to install pysolid in develop mode (editable) into the current environment
+export SETUPTOOLS_ENABLE_FEATURES="legacy-editable"
+python -m pip install -e PySolid
```

Comment:
Current support for editable/develop installs via `pyproject.toml` is beta and doesn't work for PySolid unless you export this environment variable (try it without!). It's kind of an annoyance but hopefully will go away soon. Another option is to not switch to `pyproject.toml` yet

Reply:
build from source is for advanced users, so setting an env var sounds no trouble to me at all.